src/jit/codegenarmarch.cpp

```diff
@@ -1968,6 +1964,8 @@ void CodeGen::genJmpMethod(GenTreePtr jmp)
             continue;
         }
 
+        const unsigned varNum = compiler->lvaTable.GetLclNum(varDsc);
+
```

Comment:
If you left the iteration alone (i.e., old style), you wouldn't need to do this. So is it worth it to change the iteration?